video/out/wayland_common.c

```diff
@@ -2097,6 +2103,16 @@ static bool create_input(struct vo_wayland_state *wl)
     return 0;
 }
 
+static void xdg_activate(struct vo_wayland_state *wl)
+{
+    const char *token = getenv("XDG_ACTIVATION_TOKEN");
+    if (token) {
+        MP_VERBOSE(wl, "Activating window with token: %s\n", token);
```

Comment:
```suggestion
        MP_VERBOSE(wl, "Activating window with token: '%s'\n", token);
```
Nit but that way the empty string case would show up as `''`.

Reply:
On that note, wouldn't be better to check for that? `if (token && token[0]) {`? Does it make sense to pass empty string to `xdg_activation_v1_activate`?

Reply:
Empty tokens are allowed by the protocol.

Reply:
Do we know that an empty token is invalid? If we do, then it does make sense, but otherwise not IMHO.

Imagine a compositor which decides to bilindly give this ability to clients by passing an empty string, for instance.

Reply:
> Empty tokens are allowed by the protocol.

Clear. It's correct in this case.